test/errors-specs.js

```diff
@@ -0,0 +1,107 @@
+import { errors } from '../lib/errors.js';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.use(chaiAsPromised);
+
+// Error codes and messages have been added according to JsonWireProtocol see
+// https://code.google.com/p/selenium/wiki/JsonWireProtocol#Response_Status_Codes
+let errorsList = [
+  { errorName: 'NoSuchDriverError',
```

Comment:
same thing with spaces and braces here. these are plain js objects, so no space after brace (that's just for import/export as a way to differentiate them)